modules/soplex/7.1.2.bcr.1/overlay/BUILD.bazel

```diff
@@ -0,0 +1,46 @@
+# Description:
+# Soplex is a LP solver, a common tool in Operations Research.git
+
+cc_library(
+    name = "libsoplex",
+    srcs = glob(
+        ["src/soplex/*.cpp"],
+        exclude = ["src/git_hash.cpp"],
+    ),
+    hdrs = glob(
+        [
+            "src/soplex/*.h",
+            "src/soplex/*.hpp",
+        ],
+    ) + [
+        "src/soplex.h",
+        "src/soplex.hpp",
+        "src/soplex/git_hash.cpp",
+    ],
+    copts = select({
+        "@platforms//os:windows": ["-utf-8"],
+        "//conditions:default": [],
+    }) + [
+        "-Isrc/soplex/external",
```

Comment:
This won't be the correct path if the module is used as a dependency (vs. as the root module). You can move this to a separate target, use `includes` and depend on it via `implementation_deps`.